### DISCUSSION TITLE
Manually ordering read lists

### DISCUSSION THREAD

**Discussion by crank8877** [gotson/komga#1781] [Q&A]
Is there another way to manually order the read lists than dragging to reposition? I've got a massive number of comics (500+) from across different series, and dragging to reposition would take forever. Or could I create my own reading list? If so, what is needed for this? The documentation says the following about information a CBL contains:

```
read list name
books contained in the list, in order
for each book:
name of the series it is part of
volume of the series
year of the series
number of that book in the series
```

But I don't know how to put this in a CBL. Do I create a spreadsheet with each of this info in columns and change the extension to CBL? I've looked at other reading lists, but they have an "id" field and I don't know what this references.

Any help would be great.

**Comment by gotson** [comment]
> Is there another way to manually order the read lists than dragging to reposition?

Not from the webui. See #1584

**Reply by crank8877** [reply]
I thought as much. So seems the best thing to do would be to create my own CBL. Which brings me back to the second portion of the question, is there a guide somewhere for how to do this?

**Reply by gotson** [reply]
CBL is from ComicRack. You might better use the Komga API directly.

**Reply by crank8877** [reply]
Forgive my ignorance, I’m completely new to all this, but how would I do that?

**Reply by gotson** [reply]
Komga has a [REST API](https://komga.org/docs/api/rest). If you know nothing about this then maybe the solution is not suited for you. You might better join the Discord server and ask for advice there.

**Reply by crank8877** [reply]
Definitely over my head. I'll try the Discord then, thanks.
